balena/models/device.py

```diff
@@ -1424,8 +1424,8 @@ def start_os_update(self, uuid, target_os_version):
             InvalidParameter|OsUpdateError: if target_os_version is invalid.
 
         Examples:
-            >>> balena.models.device.start_os_update('b6070f4fea5edf808b576123157fe5ec', '2.29.2+rev1.prod')
-            {u'status': u'in_progress', u'action': u'resinhup', u'parameters': {u'target_version': u'2.29.2+rev1.prod'}, u'last_run': 1554490809219L}
+            >>> balena.models.device.start_os_update('b6070f4fea5edf808b576123157fe5ec', '2.29.2+rev1')
```

Comment:
Let's add the variant back to the example so that's more obvious that we expect the fully qualified OS version, like:
```suggestion
            >>> balena.models.device.start_os_update('b6070f4fea5edf808b576123157fe5ec', '2.29.2+rev1.prod')
            >>> # or for unified OS releases
            >>> balena.models.device.start_os_update('b6070f4fea5edf808b576123157fe5ec', '2.89.0+rev1')
```